nimble/host/mesh/src/testing.c

```diff
@@ -171,7 +171,9 @@ void bt_test_print_credentials(void)
 		console_printf("Friend cred: %d\n", i);
 		console_printf("\tNetKeyIdx: %04x\n",
 			           sub[i].net_idx);
-		console_printf("\tNID: %02x\n", subnet->keys->msg.nid);
+		if (subnet) {
+            console_printf("\tNID: %02x\n", subnet->keys->msg.nid);
+        }
```

Comment:
What about the next line? And indentation.

Reply:
done